src/services/impl/SanitizerImpl.ts

```diff
@@ -0,0 +1,14 @@
+import {Sanitizer} from "../Sanitizer";
+
+export class SanitizerImpl implements Sanitizer {
+
+  validateEmail(email?: string): string | null {
+    const regex = /(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9]))\.){3}(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9])|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])/;
+    return email?.match(regex) ? email : null;
+  }
+
+  validateMobilePhone(mobilePhone?: string): string | null {
```

Comment:
If we would like to validate a mobilePhone, it means we receive one in input. mobilePhone should not be nullable from my point of view